deployment/provision_vagrant_vm.sh

```diff
@@ -11,9 +22,13 @@ export DEBIAN_FRONTEND=noninteractive
 
 # install python stuff
 apt-get -q update
-apt-get -q install -y python3.7 python3.7-dev python3-venv python3.7-venv gettext
+apt-get -q install -y sudo python3.7 python3.7-dev python3-venv python3.7-venv gettext git wget npm
 
 # setup postgres
+if [ "$docker" = true ]
```

Comment:
If [this article](https://www.cyberciti.biz/faq/how-to-declare-boolean-variables-in-bash-and-use-them-in-a-shell-script/) didn't lie to me, this should be
```suggestion
if "$docker"
```